### DISCUSSION TITLE
🌍 Localization Launch

### DISCUSSION THREAD

**Discussion by fuksman** [anyproto/contributors#45] [Announcements]
Dear community, we're delighted to announce the launch of the Localization project!

We have started accepting translations for Anytype Desktop. If you have ever dreamt of using Anytype in your native language, now you can contribute, and we will include your translations in upcoming releases.

You can participate via the [Crowdin platform](https://crowdin.com/project/anytype-desktop).

If your language is not listed on Crowdin yet, please leave a comment here, and we will swiftly add it.

UPD: we started to collect answers about translation policy in our docs: [https://tech.anytype.io/l10n/policy](https://tech.anytype.io/l10n/policy).

While we are working on optimizing Anytype Desktop for right-to-left languages, we do not currently support their translation. 
Stay tuned 🙌

Our next steps:
-  launching localization for Anytype Mobile
-  optimizing apps for RTL languages

**Comment by ahmedwael216** [comment]
Please, add arabic 🫶.

**Reply by fuksman** [reply]
Will do as soon as RTL is supported

**Reply by aadimator** [reply]
Also, kindly add Urdu as well, when RTL is supported. I'd like to contribute. Thanks

**Reply by Deepakshrivas07** [reply]
great
❤️

**Reply by AmmarHaddadi** [reply]
someone ping me when RTL is supported so I can help translating to arabic

**Comment by chrtz** [comment]
Some sort of language style guide would be nice/helpful. Like to use informal or formal form of address (e.g. "du" or "sie" in german) and a general description of preferred tone of voice.

**Reply by fuksman** [reply]
Great thought! We will address this.

**Reply by fuksman** [reply]
@chrtz, we discussed this internally, and the answer is that the tone should be informal. I've published some pages related to localization on tech.anytype.io. Here is a policy: [https://tech.anytype.io/l10n/policy](https://tech.anytype.io/l10n/policy).

The documentation is open, and everyone is welcome to contribute there as well: [https://github.com/anyproto/tech-docs](https://github.com/anyproto/tech-docs).

**Comment by bonnyflo** [comment]
Could you please add Hungarian as well? I would be happy to translate it!

**Reply by fuksman** [reply]
Done ✅

**Comment by ghost** [comment]
Please add Greek 🙏

**Reply by fuksman** [reply]
Done ✅

**Comment by sudanchapagain** [comment]
Add Nepali too.

**Reply by fuksman** [reply]
Done ✅

**Comment by jponican** [comment]
Slovak, please.

**Reply by fuksman** [reply]
Done ✅

**Reply by jponican** [reply]
Thanks!

**Comment by pavlozaikin** [comment]
ANY should follow some in-app language support like Slack

**Reply by fuksman** [reply]
Hey @paulelpr, could you please provide more details here? What do you suggest implementing?

**Reply by pavlozaikin** [reply]
Some languages that have been added to the apps are not currently available. If someone wants to use it, they can install it in a third-party way (for example, using a translator).

**Reply by fuksman** [reply]
Ok, I got it. Of course we will provide an option to chose the language via app settings since our next release (planned for Aug 15). We will include all languages with sufficient translations (I think it will be everything with >80% progress).

**Reply by pavlozaikin** [reply]
It's not just about choosing a language in the settings. It's also about excluding certain languages (no translation, technical ability to display, loss of reputation of a certain language) 🙂

I think we understand each other

**Reply by Dorifor** [reply]
> It's also about excluding certain languages 

I have to admit I don't really understand what you meant, you want to exclude languages or the opposite ? (English is not my mother tongue 😅)

**Comment by jjezewski** [comment]
Please add Polish.

**Reply by fuksman** [reply]
Done ✅

**Reply by jjezewski** [reply]
Thanks 👍

**Comment by aidilumarov** [comment]
Could you please add Kyrgyz?

**Reply by fuksman** [reply]
Done ✅

**Comment by miaoziguan** [comment]
Could you please add Chinese?

**Reply by fuksman** [reply]
It is already there in simplified and traditional versions.

**Comment by smutevelli** [comment]
Could you please add Turkish?

**Reply by fuksman** [reply]
Done ✅

**Comment by pizofreude** [comment]
Can you kindly add Malay?
Malaysia 🇲🇾 
Cheers!

**Reply by fuksman** [reply]
Done ✅

**Comment by sys-t-m** [comment]
Hi Sergey, can you kindly add Armenian? 
Thanks in advance!

**Reply by fuksman** [reply]
Added 🙌

**Reply by sys-t-m** [reply]
Thanks a lot!

**Comment by kersmila** [comment]
Filipino (Tagalog), please.

**Reply by fuksman** [reply]
Done ✅

**Comment by psyaito** [comment]
Japanese please? Would love to translate it

**Reply by fuksman** [reply]
Done ✅

**Comment by Pedaleiro** [comment]
Plese, could you add pt-BR, Brazilian Portuguese?

**Reply by fuksman** [reply]
It's already there

**Comment by kovica** [comment]
Can you, please, also add Slovenian (sl_SI) language?

**Reply by fuksman** [reply]
Done ✅

**Comment by nhandoduc** [comment]
Please, Could you add Vietnamese. Thank you!

**Reply by fuksman** [reply]
Done ✅

**Comment by hmamigo** [comment]
Español, por favor.

**Reply by fuksman** [reply]
It's already there and translated for 100% 👌

**Reply by hmamigo** [reply]
Thank you so much! :+1:

**Comment by Shampra** [comment]
If there are several translations, how do you choose the right one?
I have a case where the last one proposed (and therefore integrated into AT) is bad. If I vote for the other one, which is correct, will it be integrated?
(in Crowdin)

**Reply by fuksman** [reply]
We don't have the ability to choose the correct translation; therefore, we rely on the community's crosscheck here. You can vote for translations or suggest new ones, and we will update localization files before the release.
Thanks for the participation!

**Reply by Shampra** [reply]
Thanks :)

@fuksman, there are translation updates that add things to the English version, updates that come from "Any association".
Does this come from the team? Is there a reason?
On the screenshot, it adds a "?" that opens a popup explaining how to save your recovery phrase, which is a bit out of context... 

If it's a choice of the AT team, I'm reluctant to downvote.

![image](https://github.com/anyproto/contributors/assets/16141040/c7e48ab5-88f9-4af4-b7ea-451d9c73a92b)

**Reply by fuksman** [reply]
@Shampra, I have renamed my personal account from @fuksman to @any-association on Crowdin today. So, if you see @any-association as the author of translations, it simply means that they were automatically imported from GitHub (where other users could suggest them via PRs). Our team only supports English and does not participate in translations to other languages. 

TL;DR: feel free to modify any translations that appear to be made by @any-association.

**Reply by Shampra** [reply]
Ok, thank you, let's go 😁

**Comment by HeekangPark** [comment]
Korean please. I can contribute translation.

**Reply by fuksman** [reply]
@HeekangPark it's already there, welcome :)

**Comment by blacklightpy** [comment]
Add Malayalam please!

**Reply by fuksman** [reply]
Done ✅

**Comment by holton-jiang** [comment]
I have finished translating the String file on iOS. Can the developer please merge it, thanks.😁

- [x] Simplified Chinese translation

https://github.com/anyproto/anytype-swift/pull/308

https://github.com/anyproto/anytype-swift/pull/309

**Comment by JonathanReeve** [comment]
Esperanto, please. If necessary I can help with the translation, or provide references for translators.

**Reply by fuksman** [reply]
Done ✅

**Comment by ghost** [comment]
Please add Arabic.

**Reply by fuksman** [reply]
Will do as soon as RTL is supported

**Comment by birukbelay** [comment]
can you add Amharic, i will be glad to help

**Reply by fuksman** [reply]
Done ✅

**Comment by lostopkk** [comment]
add website translation feature.

**Comment by DevCode-shreyas** [comment]
Can you add the Marathi language, I will be glad to help.

**Reply by fuksman** [reply]
Done ✅

**Comment by aswanijehangeer** [comment]
Can you add the Sindhi language, I will be glad to help :)

**Reply by fuksman** [reply]
Will do as soon as RTL is supported

**Comment by Swarga-codes** [comment]
Please add Gujarati.

**Reply by fuksman** [reply]
Done ✅

**Comment by Abhi-1631** [comment]
Please add Hindi

**Reply by fuksman** [reply]
The Hindi translation has been available since the beginning of the l10n project.

**Comment by AbdulRehmanGHub** [comment]
Nice steps, please add Urdu also

**Reply by fuksman** [reply]
Will do as soon as RTL is supported

**Comment by AcademiccMacharia** [comment]
Kindly add Swahili

**Reply by fuksman** [reply]
Done ✅

**Comment by Naveenchand06** [comment]
Kindly add Tamil

**Reply by fuksman** [reply]
Done ✅

**Comment by orenk86** [comment]
Please add Hebrew, I'd love to help

**Reply by fuksman** [reply]
Will do as soon as RTL is supported

**Comment by enochmakafui** [comment]
Add Ewe

**Reply by fuksman** [reply]
Done ✅

**Comment by lscherub** [comment]
Hi there, can we please add support for Punjabi?

**Reply by fuksman** [reply]
Done ✅
